frontend/src/metabase/core/components/Slider/Slider.tsx

```diff
@@ -18,76 +22,121 @@ export type NumericInputAttributes = Omit<
 >;
 
 export interface SliderProps extends NumericInputAttributes {
-  value: number[];
-  onChange: (value: number[]) => void;
+  value: (number | undefined)[];
+  onChange: (value: (number | undefined)[]) => void;
   min?: number;
   max?: number;
   step?: number;
 }
 
 const Slider = ({
-  value,
+  value: parentValue,
   onChange,
-  min = 0,
-  max = 100,
+  min: parentMin = 0,
+  max: parentMax = 100,
   step = 1,
 }: SliderProps) => {
-  const [rangeMin, rangeMax] = useMemo(
-    () => [Math.min(...value, min, max), Math.max(...value, min, max)],
-    [value, min, max],
-  );
+  const [isHovering, setIsHovering] = useState(false);
+  const [value, setValue] = useState([
+    parentValue[0] ?? parentMin,
+    parentValue[1] ?? parentMax,
+  ]);
 
-  const [beforeRange, rangeWidth] = useMemo(() => {
-    const totalRange = rangeMax - rangeMin;
+  // potentially expand the input range to include out-of-range values
+  const [min, max] = useMemo(() => {
+    return [_.min([...value, parentMin]), _.max([...value, parentMax])];
```

Comment:
the underscore `min` and `max` functions handle `undefined` values better than `Math.min` and `Math.max`

Reply:
Haha, it's actually surprising the difference between the two (just gave it a try). Would it just be a matter of filtering out undefined from `value` though to use `Math.min`?

Reply:
yeah, i actually did that in a prior version, but this is much easier to read.